### ISSUE TITLE
Acts: Judgement of Israel

### ISSUE THREAD

**Issue by sjanhunen** [sjanhunen/dokimazo#14]
During the second major arc, we see the following points:

1. Writing of the Covenant
2. Judgment of the Idolatrous
3. Selection of the Faithful

Thesis: to fully understand Acts (the offering of the NT) we need to understand the camp at Sinai in Exodus  (the offering of the OT). See #6.

**Comment by sjanhunen** [comment]
Where does Penticost fit within the Exodus story? Something of significance: Moses was up in the mountain for 40 days (Exo 24:18) in association with the presentation of the Old Covenant. The disciples were with Christ 40 days (Acts 1:3) getting instruction concerning the "kingdom" and the New Covenant.

**Comment by sjanhunen** [comment]
Penticost, then, is the presentation of the New Covenant and Kingdom. Acts follows the Apostles as God repeatedly shows grace and patience with the nation of Israel.

What parallels can we draw between the Acts period and the Sinai period? See Paul's teaching on Sinai and Jerusalem (Gal 4:24-25).

It is critical to see that the writing of the Covenant followed the blood of the covenant. In the case of the Old Testament, the writing was on tablets on stone. But in the case of the New Covenant, the writing was on the heart:

[Jer 31:33 NKJV] 33 "But this [is] the covenant that I will make with the house of Israel after those days, says the LORD: I will put My law in their minds, and write it on their hearts; and I will be their God, and they shall be My people.

This is the great contrast between the Letter and the Spirit that Paul discusses.

[2Co 3:6 NKJV] 6 who also made us sufficient as ministers of the new covenant, not of the letter but of the Spirit; for the letter kills, but the Spirit gives life.

**Comment by sjanhunen** [comment]
Notes on giving of the law and Penticost: https://www.biblestudyproject.org/bible-study-library/israel/was-the-law-of-moses-given-on-pentecost-a-tale-of-two-occasions/

**Comment by sjanhunen** [comment]
It's important to realize that a nation or kingdom cannot be established without an agreement between the King and the people.

**Comment by sjanhunen** [comment]
Also of significance is the fact that God gave detailed instructions on the first tabernacle in Exo 25. Think about how God is preparing the Heavenly Tabernacle (New Jerusalem) through the ministry of Paul.

**Comment by sjanhunen** [comment]
Mapping the sequence of events at the Old Covenant helps us understand the New Covenant. The blood was shed first, then Moses ascended into the mountain, and the 40 days of instruction followed.

**Comment by sjanhunen** [comment]
Another parallel worth examining: the trumpet in Exo 19:16-20 at the establishment of the Old Covenant and the trumpets in Revelation. See also parallels in Hebrews 12:18-24.

**Comment by sjanhunen** [comment]
One most important realization to have is that the believers during the Acts period expected the imminent return of Christ and His Kingdom. It appears they were living a life in expectation of the full realization of the New Testament (Covenant). However, we only see the firstfruits of this in Acts. They don't get out of the camp at Sinai, so to speak.

**Comment by sjanhunen** [comment]
Are there parallels to draw between the multiple trips Moses made up and down the mountain and the work of Christ in the Gospels and Acts period. How many times did Christ ascend and descend? Recall that it was necessary for Him to first ascend to His father in John 20:17. This is a first ascention. But Christ was witnessed by others after this, much like the seventy saw Jehovah in Exodus 24:9-11.

**Comment by sjanhunen** [comment]
Moses had the closest encounter with Jehovah after he broke the first set of stone tablets and returned again to the mountain (Exodus 33). Is Christ still "in the mountain" until His return?

**Comment by sjanhunen** [comment]
A key point in any case is that ultimately entering into the Kingdom in the OT required the journey across the wilderness and the victory at Jerico. These events in the NT are future.

**Comment by sjanhunen** [comment]
In Acts, the wedding feast hasn’t taken place yet. And Paul refers to believers as “espoused” in Corinthians.

**Comment by sjanhunen** [comment]
Parallels between Exodus and Acts: http://acts242study.com/they-many-parallels-of-sinai-and-pentecost/

Need to study in more depth when the law was given with respect to penticost.

**Comment by sjanhunen** [comment]
The journeys Moses took up and down the mountain are significant. In particular, the return for judgment on the camp (breaking the tablets) and the return with a shining face to lead the people. Think about how Jehovah was interceding for Israel. (Exo 32-34).

**Comment by sjanhunen** [comment]
Also significant is the tribe of Levi in Exo 32:26. They were the ones on the Lord’s side. God was working through them in the rebellious situation.

**Comment by sjanhunen** [comment]
Where are passages from this situation in Exodus quoted in Acts or the NT? Paul quotes Exo 33:19 in Rom 9:15.

**Comment by sjanhunen** [comment]
Compare the acknowledgment the people made in Exodus (“all the words which the Lord hath said we will do”) multiple times with what is going on in Acts. Was an acknowledgement like this made? What is the significance of the quoting of Isaiah multiple times in Acts about not seeing, hearing, or understanding. Moses showed and read. And the people acknowledged.

**Comment by sjanhunen** [comment]
How far does the nation of Israel get in the book of Acts? Does the golden calf scene fit into Acts, or does that appear only in the book of Revelation?

**Comment by sjanhunen** [comment]
In Exodus, after the Passover and redemption from Egypt, we have a 3-fold affirmation of the kingdom covenant between Israel and Jehovah:

* [Exo 19:7-8 KJV] 7 And Moses came and called for the elders of the people, and laid before their faces all these words which the LORD commanded him. 8 And all the people answered together, and said, All that the LORD hath spoken we will do. And Moses returned the words of the people unto the LORD.
* [Exo 24:3 KJV] 3 And Moses came and told the people all the words of the LORD, and all the judgments: and all the people answered with one voice, and said, All the words which the LORD hath said will we do.
* [Exo 24:7 KJV] 7 And he took the book of the covenant, and read in the audience of the people: and they said, All that the LORD hath said will we do, and be obedient.

It is after this oath was taken that Moses sprinkled the blood on the people. Moses specifically announced that this was the blood of the covenant. And then after this Moses and the leadership saw Jehovah and ate and drank.

**Comment by sjanhunen** [comment]
Comparing this to the Gospels and Acts, we see the following pattern:

- Jesus shares a meal with his disciples at Passover (they are eating and drinking with Jehovah)
- At this meal, Jesus breaks the bread and introduces the wine as the blood of the New Covenant (Testament) - He specifically says He won't do this again until He does it in His Kingdom 
- Jesus Himself becomes the Passover Lamb
- Penticost takes place with definite parallels with the scene at Sinai (but, did these really happen at the same time?)
- In Revelation, the marriage supper of the Lamb (Rev 19:19) is announced before the 1000 years. What other Scriptures refer to this. Is this what Christ was referring to at Passover?

**Comment by sjanhunen** [comment]
Note that three thousand were saved in Acts 2:41, while three thousand were judged and slain in Exodus 32:28.

**Comment by sjanhunen** [comment]
And how does the hope presented in Zechariah fit into this picture?

**Comment by sjanhunen** [comment]
Very interesting that Paul refers back to Jannes and Jambres resisting Moses in 2 Timothy. Looking at how many times Paul refers back to Exodus and Moses would be a study in itself!

**Comment by sjanhunen** [comment]
The Gospels also cover a good portion of the NT Exodus story (the parts that happen before passover).

**Comment by sjanhunen** [comment]
Where does the spirit fit into the picture at Penticost? Rather than Moses leaving Aaron and Hur with the people, Christ left the Holy Spirit as a guide.

**Comment by sjanhunen** [comment]
In his great speech, Stephen refers back to the calk scene from Exodus:

[Act 7:41 NKJV] 41 "And they made a calf in those days, offered sacrifices to the idol, and rejoiced in the works of their own hands.

The linkage to idols during the Acts period is striking.

**Comment by sjanhunen** [comment]
Look at the theme of preparation throughout the book of Acts. In other words, God was completing something so that when He returns, it will be ready. There was a divine purpose being worked out during Acts that we may miss if we view it as failure. In fact, God was assembling a group of faithful despite all the failure. The forming of the priestly tribe.

**Comment by sjanhunen** [comment]
Important parallels:
- Without the camp
- The faithful remnant
- Both Jew and Gentile counted as faithful
- Rebellion
- Judgment on the camp

**Comment by sjanhunen** [comment]
A question: did the tribe of Levi enter the promised land. Were they "numbered" with the other Israelites that were punished?

See Numbers 13:1-17.

Levi was already excluded from the mix here.

**Comment by sjanhunen** [comment]
http://stepintothestory.ca/12-spies-sent-to-investigate-the-promised-land/

**Comment by sjanhunen** [comment]
The Sinai camp period was pivotal in sifting out the faithful believers from all those that were redeemed from Egypt. If we go forward to the entrance to the promised land, very few mentioned in Exodus actually made it across the Jordan: only Joshua and the tribe of Levi. Levi had no inheritance in the Land. Caleb made it too, but he is not mentioned until the spies are dispatched in Numbers 13:6.

**Comment by sjanhunen** [comment]
These was a second selection period after all the battles with Kings. In this case, 12 spies were sent out to scope out the promised land. Only two of the 12 that came back faithful to the Lord: Joshua and Caleb.

**Comment by sjanhunen** [comment]
It seems that the "selection" period of faithful Israel is limited to Sinai and the Jordan crossing.

**Comment by sjanhunen** [comment]
The restriction on who could enter the promised land was based upon those that were Numbered with Israel (Numbers 14:29). Levi was not numbered with Israel (Num 1:49-54). Levi had a special purpose within the Nation.

[Num 1:49-50 NKJV] 49 "Only the tribe of Levi you shall not number, nor take a census of them among the children of Israel; 50 "but you shall appoint the Levites over the tabernacle of the Testimony, over all its furnishings, and over all things that belong to it; they shall carry the tabernacle and all its furnishings; they shall attend to it and camp around the tabernacle.

**Comment by sjanhunen** [comment]
Theory: God was selecting and preparing the "Faithful Levites" of the New Covenant during Acts.

**Comment by sjanhunen** [comment]
An interesting read on the Sabbath as the sign of the Law: http://www.sabbatismos.com/the-sabbath/sign-of-the-mosaic-covenant/#sthash.vvRrFRWp.u9LqnJx5.dpbs

**Comment by sjanhunen** [comment]
The tablets of stone brought down from the mountain were written by the finger of God. This is different than having the law written in the heart.

[Jer 31:33 KJV] 33 But this [shall be] the covenant that I will make with the house of Israel; After those days, saith the LORD, I will put my law in their inward parts, and write it in their hearts; and will be their God, and they shall be my people.

So the writing of God is a definite parallel. Other sign gifts go along with this. Paul specifically speaks of this writing in the heart as evidence:

[2Co 3:2-3 KJV] 2 Ye are our epistle written in our hearts, known and read of all men: 3 [Forasmuch as ye are] manifestly declared to be the epistle of Christ ministered by us, written not with ink, but with the Spirit of the living God; not in tables of stone, but in fleshy tables of the heart.

**Comment by sjanhunen** [comment]
Was not Sinai the Judgement of Israel? God Judged His own Nation first before he went out and laid down the judgements on the other nations around them. Peter says something interesting about this:

[1Pe 4:17 KJV] 17 For the time [is come] that judgment must begin at the house of God: and if [it] first [begin] at us, what shall the end [be] of them that obey not the gospel of God?

Is it then not true that Judgement began with the house of Israel during Acts? This must happen before the Judgements are brought down to the other Nations in Revelation.

**Comment by sjanhunen** [comment]
The tablets were for teaching and instruction:

[Exo 24:12 KJV] 12 And the LORD said unto Moses, Come up to me into the mount, and be there: and I will give thee tables of stone, and a law, and commandments which I have written; that thou mayest teach them.

The Spirit given at Penticost would lead into truth.

**Comment by sjanhunen** [comment]
Is the shift from Preach to Proclaim significant from the Gospels to Acts?

**Comment by sjanhunen** [comment]
When the Letter came down the mountain in Exodus, the result was the death of 3000. When the Spirit came down at Penticost, 3000 were saved.

**Comment by sjanhunen** [comment]
Look at the theme of God as a consuming fire in light of Judgement in Exodus and Acts.

**Comment by sjanhunen** [comment]
The Blessing given to Levi as compared to the Blessing of Abraham. The blessing of the faithful.

**Comment by sjanhunen** [comment]
Paul goes through the judgements that were handed out when Jehovah was not pleased with the people during Exodus:

[1Co 10:1, 5, 7-8 NKJV] 1 Moreover, brethren, I do not want you to be unaware that all our fathers were under the cloud, all passed through the sea, ... 5 But with most of them God was not well pleased, for [their bodies] were scattered in the wilderness. ... 7 And do not become idolaters as [were] some of them. As it is written, "The people sat down to eat and drink, and rose up to play." 8 Nor let us commit sexual immorality, as some of them did, and in one day twenty-three thousand fell;

The whole chapter is an important reading on this. It also shows us the true context of "temptation" or "testing" going back to Exodus.

**Comment by sjanhunen** [comment]
A great question here is who was doing the punishment? Moses or Jehovah. Moses cast down the idol, but the Lord judged many by the sword and the plague. Jehovah was angry with Israel, but Moses interceded  for the nation. However, Jehovah reserved punishment for a later time.

[Exo 32:34 NKJV] 34 "Now therefore, go, lead the people to [the place] of which I have spoken to you. Behold, My Angel shall go before you. Nevertheless, in the day when I visit for punishment, I will visit punishment upon them for their sin."


https://en.wikipedia.org/wiki/Siege_of_Jerusalem_(70_CE)

**Comment by sjanhunen** [comment]
The tribe of Levi was not to be concerned with earthly matters. They were special ministers to God. Could it be that the Blessings of Abraham under the New Covenant find an echo here? In other words, does Levi really play any role in the Battles of Revelation?

It's also important to realize that tribe of Levi in general were not specifically the priests (that belonged to Aaron and his sons). But they had special roles within the Tabernacle.

https://en.wikipedia.org/wiki/Tribe_of_Levi
http://www.israel-a-history-of.com/tribe-of-levi.html

**Comment by sjanhunen** [comment]
See City of Refuge, open to Jew and Gentile: https://cdn.pbc.org/Main_Service/1996/09/15/4471.html

Significant that they were for someone who had killed someone accidentally.

**Comment by sjanhunen** [comment]
The blessing on Exodus was given to the Levites not because they were Levites but because they were the faithful believers.
